### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by elhorton**
# Thank you for helping us improve the Azure IoT Python SDK!

## Need Support

- **Have a feature request for SDKs?** Please post it on [User Voice](https://feedback.azure.com/forums/321918-azure-iot) to help us prioritize
- **Have a technical question?** Ask on [Stack Overflow with tag "azure-iot-hub"](https://stackoverflow.com/questions/tagged/azure-iot-hub)
- **Need Support?** Every customer with an active Azure subscription has access to [support](https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request) with guaranteed response time.  Consider submitting a ticket and get assistance from Microsoft support team
- **Found a bug?** Please help us fix it by thoroughly documenting it and filing an issue on GitHub (See below).

Here's a little checklist of things that will help it make its way to the repository: Note that you don't have to check all the boxes, we can help you with that.
This being said, the more you do, the quicker it'll go through our gated build!
-->

## Checklist

- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-iot-sdk-python/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

## Description of the problem

Please be as precise as possible: what issue you experienced, how often...

## Description of the solution

How you solved the issue and the other things you considered and maybe rejected

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## Need Support
 
 - **Have a feature request for SDKs?** Please post it on [User Voice](https://feedback.azure.com/forums/321918-azure-iot) to help us prioritize
-- **Have a technical question?** Ask on [Stack Overflow with tag "azure-iot-hub"](https://stackoverflow.com/questions/tagged/azure-iot-hub)
+- **Have a technical question?** Please see [IoT support options](aka.ms/IoTHelp) for information on posting questions on Microsoft Q&A and Stack overflow using the "azure-iot-sdk" tag.
```
Comment (BertKleewein) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: >aka.ms/IoTHelp [](start = 66, length = 14)

This needs to be a full URI ([https://aka.ms/IoTHelp](https://aka.ms/IoTHelp))

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ assignees: ''
 INSTRUCTIONS
 ==========
 
-Please follow the instructions and template below to save us time requesting additional information from you. For more information on all of your support options, please see [Azure IoT support and help options](https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-support-help).
+Please follow the instructions and template below to save us time requesting additional information from you. For more information on all of your support options, please see [Azure IoT support and help options](aka.ms/IoTHelp).
```
Comment (BertKleewein) [inline .github/ISSUE_TEMPLATE/bug-report.md:15]: >a.ms/IoTHelp [](start = 213, length = 12)

add https:// here too

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: Azure IoT support and help options
-    url: https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-support-help
+    url: aka.ms/IoTHelp
```
Comment (BertKleewein) [inline .github/ISSUE_TEMPLATE/config.yml:4]: >aka.ms/IoTHelp [](start = 9, length = 14)

Don't know if this needs https:// or not, but I assume it does

File: .github/ISSUE_TEMPLATE/technical-question.md [hunk 4]
```diff
@@ -11,12 +11,14 @@ assignees: ''
...
+
+For answers on your developer questions from the largest community developer ecosystem, ask your question on Stack Overflow. We actively monitor all questions tagged wtih "azure-iot-sdk".
 
 Additionally, if your technical question requires submitting service logs, you can submit an [Azure Support Ticket](https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request).
 
 If you have a question specifically on a Microsoft page you've read, please open an issue on the document itself. This ensures you get the fastest targeted response, and you can do this by scrolling to the bottom of the page to the **Feedback** section. 
 
-For more information on the support options available to you, please see [Azure IoT support and help options](https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-support-help).
+For more information on the support options available to you, please see [Azure IoT support and help options](aka.ms/IoTHelp).
```
Comment (BertKleewein) [inline .github/ISSUE_TEMPLATE/technical-question.md:22]: >aka.ms/IoTHelp [](start = 110, length = 14)

add https://

**Comment by BertKleewein** [comment]
Signing off assuming that you're adding the https// in all the right places.

**Comment by elhorton** [comment]
> Did you check if all the aka.ms links work from outside the MS environment ?
> May be it does but I never tried it , so asking.

yes, I've double checked the links and they work. also added the full URI, thanks for the review

**Pull request merged**